src/main/java/lotto/domain/RandomLottoGenerator.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static lotto.domain.Lotto.LOTTO_NUMBER_COUNT;
+import static lotto.domain.LottoNumber.LOTTO_MAX_NUMBER;
+import static lotto.domain.LottoNumber.LOTTO_MIN_NUMBER;
+
+public class RandomLottoGenerator implements LottoGenerator {
+    private final long automaticIssueQuantity;
+
+    public RandomLottoGenerator(long automaticIssueQuantity) {
+        this.automaticIssueQuantity = automaticIssueQuantity;
+    }
+
+    @Override
+    public List<Lotto> generate() {
+
+        ArrayList<Lotto> lottos = new ArrayList<>();
+
```

Comment:
별 다른 이유가 없으면 인터페이스를 사용하는 것이 좀 더 유연할 것 같아요 구현체가 변경되면 해당 부분만 변경하면 되기 때문이에요. 

```suggestion

        List<Lotto> lottos = new ArrayList<>();

```